bigquery/tests/unit/test_job.py

```diff
@@ -1753,6 +1753,24 @@ def _verifyResourceProperties(self, job, resource):
         else:
             self.assertIsNone(job.destination_encryption_configuration)
 
+        if "destinationTableDescription" in config:
```

Comment:
Sorry, `_verifyResourceProperties` isn't a very good example to follow. I prefer explicitly setting the values you are looking for in the test itself.

Please remove this code and add check for string values for `destination_table_description` and `destination_table_friendly_name` to `test_ctor_w_config`.